docs/introduction/get-started/p2p-private.md

```diff
@@ -36,7 +36,7 @@ In this section, we show you how to make private transactions and send funds to
     To do this, run:
 
     ```shell
-    miden-client tx new p2id --note-type private <regular-account-id-A> <regular-account-id-B> <faucet-account-id> 50 
+    miden-client tx new p2id --note-type private --sender <regular-account-id-A> --target <regular-account-id-B> --faucet <faucet-account-id> 50 
```

Comment:
Comment for the future: I wonder if we should come up with some format for assets. Maybe this should be something like `--asset faucetId::amount`.

Reply:
Yep - that would be even better, but a bit more complicated (since token symbols are not unique, a symbol may map to multiple faucet IDs).

Another thing we'll need to solve is how to handle non-fungible asset encoding (that one is full 32 bytes in size).

Reply:
@bobbinth @mFragaBA sorry that was probably confusing. `next` would have been the right target. 

Currently, the `main` version of the client doesn't support those changes. But they are on the live docs now. Can we revert this commit and merge against main?

Or should I open another PR against `main` to restore the old version?